src/main/com/marginallyclever/makelangelo/paper/PaperMenuItem.java

```diff
@@ -0,0 +1,26 @@
+package com.marginallyclever.makelangelo.paper;
+
+import javax.swing.*;
+
+public class PaperMenuItem extends JRadioButtonMenuItem {
+
+    private final PaperSize paperSize;
+    private final PaperSettings paperSettings;
+
+    public PaperMenuItem(PaperSettings paperSettings, PaperSize paperSize) {
+        super(paperSize.toString());
+        this.paperSettings = paperSettings;
+        this.paperSize = paperSize;
+        addActionListener((e)-> {
+            paperSettings.changePaperSize(paperSize);
+        });
+        updateSelected();
```

Comment:
right now the actionlistener fires because updateSelected is called after the listener is installed.  The effect is to set the paper size to the value to itself, which is redundant.  It's not harmful, so i say leave it.

Reply:
good point, I just removed it